src/lkl/posix-host.c

```diff
@@ -756,6 +757,16 @@ static void host_free(void *ptr)
     oe_free(ptr);
 }
 
+/**
+ * Returns the number of CPU cores to be displayed in cpuinfo
+ */
+unsigned int sgxlkl_cpuinfo_get_cpu_cores(void)
```

Comment:
As I commented in the other PR, I think this should provide the complete `/proc/cpuinfo` string.

Reply:
Done.